spec/controllers/account/tasks_controller_spec.rb

```diff
@@ -3,20 +3,35 @@
 RSpec.describe Account::TasksController, type: :controller do
   render_views
 
-  context 'PUT /:move' do
-    let!(:user) { create(:user) }
-    let!(:workspace) { create(:workspace, user_id: user.id) }
-    let!(:project) { create(:project, workspace_id: workspace.id) }
-    let!(:task) { create_list(:task, 5, project_id: project.id) }
-
-    before do
-      sign_in user
-      user.update(projects: [project])
+  let!(:user) { create(:user) }
+  let!(:workspace) { create(:workspace, user: user) }
+  let!(:project) { create(:project, workspace: workspace) }
+  let!(:task) { create_list(:task, 5, project: project) }
+
+  before do
+    sign_in user
+    user.update(projects: [project])
+  end
+
+  after do
+    sign_out user
+  end
+
+  context 'GET /projects/:project_id/task/:id' do
+    it 'should show task page' do
+      get :show, params: { project_id: project.id, id: task.first.id }
+      expect(response).to be_successful
```

Comment:
be_successful? does it work?

Reply:
DEPRECATION WARNING: The success? predicate is deprecated and will be removed in Rails 6.0. Please use successful? as provided by Rack::Response::Helpers. (called from block (3 levels) in <top (required)> at /home/bogdan/Documents/Workspace/asana/spec/controllers/account/tasks_controller_spec.rb:30